pytentiostat/plotter.py

```diff
@@ -41,6 +53,24 @@ def plot_initializer(config_data):
 
 
 def plot_updater(config_data, data, line):
+    """
+    Updates the plot based on the read/write in tester.
+
+    Parameters
+    ----------
+    config_data : dict
+        Parameters from the config file
+        
```

Comment:
remove blank lines between parameters here.  The indentation keeps the structure clear

Reply:
Done!